### PR TITLE
Migrates MobileGuide, FabricGuide, OfflineGuide from Wiki to Documentation

### PR DESCRIPTION
**Pull request by sanjeetsuhag**
As a follow-up to https://github.com/CesiumGS/cesium/pull/10638, and part of the overall effort to review and revise our documentation, this PR migrates the following guides:

- [wiki/Mobile Details](https://github.com/CesiumGS/cesium/wiki/Mobile-Details) ➡ [Documentation/Contributors/MobileGuide](https://github.com/CesiumGS/cesium/tree/wiki-migration/Documentation/Contributors/MobileGuide) 
- [wiki/Offline Guide](https://github.com/CesiumGS/cesium/wiki/Offline-Guide) ➡ [Documentation/OfflineGuide](https://github.com/CesiumGS/cesium/tree/wiki-migration/Documentation/OfflineGuide)
- [wiki/Fabric](https://github.com/CesiumGS/cesium/wiki/Fabric) ➡ [Documentation/FabricGuide](https://github.com/CesiumGS/cesium/tree/wiki-migration/Documentation/FabricGuide)

For ease of review, first few commits are checking in the docs as is, followed up commits with updates for each individual guide.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @sanjeetsuhag!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/wiki-migration/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by ggetz** [comment]
> is this because we want to be explicit about what's part of the Cesium API? Or to allow copy-pasting into Sandcastle?

Mainly, I would say to be consistant with the rest of our documentation. I do think these two reasons are a good reason to keep it as it is. In the build guide documentation, as well as in several blog posts, we do recommend not using IIFE unless it's for a simple use case despite the fact that a lot of our documentation uses the style.

File: Documentation/Contributors/MobileGuide/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Mobile Guide
+
+## Supported mobile platforms
+
+Cesium relies on WebGL, which is supported on [nearly all major mobile hardware platforms and browsers](https://caniuse.com/webgl). However, WebGL may be unavailable on some older devices.
```
Comment (ggetz) [inline Documentation/Contributors/MobileGuide/README.md:5]:
```suggestion
CesiumJS relies on WebGL, which is supported on [nearly all major mobile hardware platforms and browsers](https://caniuse.com/webgl). However, WebGL may be unavailable on some older devices.
```

**Comment by ggetz** [comment]
Thanks @sanjeetsuhag! Just one remaining Cesium -> CesiumJS.

**Comment by ggetz** [comment]
Thanks @sanjeetsuhag!

**Pull request merged**